hooks/useManageToken.tsx

```diff
@@ -129,7 +129,7 @@ const useManageToken = () => {
   const rowData = useMemo(() => {
     if (activeWallet) {
       return activeWallet['created-assets']
-        .filter((as) => !as.deleted)
+        .filter((as) => !as.deleted && as.index.toString().includes(formData.assetId))
```

Comment:
This is a good start but the filtering method should  filter from the beginning of `formData.assetId` so `includes()` is not the right method. When a user types an assetId into the form they are expecting the filter to list assetId's which first character matches their input.

Reply:
"When a user types an assetId into the form they are expecting the filter to list assetId's which first character matches their input." -- this is exactly what includes does. Did you test?

Reply:
Ok, will update that